.github/workflows/build.yml

```diff
@@ -107,6 +107,7 @@ jobs:
       matrix:
         scenario:
           - default
+          - systemd-enabled
```

Comment:
I think I would prefer the scenario to be named `systemd_enabled` as we have preferred underscores to hyphens in other Ansible roles.

Reply:
The scenario and all associated variables have been changed to `systemd_enabled`